src/test/java/com/github/underscore/CollectionsTest.java

```diff
@@ -90,16 +90,16 @@ public void apply(Integer item) {
     @Test
     public void forEach() {
         final List<Integer> result = new ArrayList<Integer>();
-        $.forEach(asList(1, 2, 3), new Block<Integer>() {
-            public void apply(Integer item) {
+        $.forEach(asList(1, 2, 3), new Consumer<Integer>() {
+            public void accept(Integer item) {
                 result.add(item);
             }
         });
         assertEquals("[1, 2, 3]", result.toString());
         final List<Map.Entry<String, Integer>> resultChain = new ArrayList<Map.Entry<String, Integer>>();
         $.chain((new LinkedHashMap<String, Integer>() { { put("a", 1); put("b", 2); put("c", 3); } }).entrySet())
-            .forEach(new Block<Map.Entry<String, Integer>>() {
-                public void apply(final Map.Entry<String, Integer> item) {
+            .forEach(new Consumer<Map.Entry<String, Integer>>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.forEach\(Consumer\) ignored, is this OK in com\.github\.underscore\.CollectionsTest\.forEach\(\)